TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot_components/robot/Robot.java

```diff
@@ -48,6 +50,7 @@ public Robot(HardwareMap hardwareMap, Telemetry telemetry) {
         extenderMotor.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
         dropperServo = hardwareMap.get(Servo.class, "dropperServo"); //need testing
         spinnerServo = hardwareMap.get(CRServo.class, "spinnerServo");
+        spinnerMotor = hardwareMap.get(DcMotor.class, "odo");
```

Comment:
this is because we were using a motor on the same port as the odometry- it will get removed when the spinner is fixed but thats a different task